containers/python/TensorFlowCifarDockerfile

```diff
@@ -2,7 +2,7 @@ FROM clipper/py-rpc:latest
 
 MAINTAINER Dan Crankshaw <dscrankshaw@gmail.com>
 
-RUN pip install tensorflow
+RUN pip install tensorflow==1.0
```

Comment:
This is to make sure that the tensorflow version is compatible with our pre-trained model